utils/HandleUntrustedRequests.md

```diff
@@ -0,0 +1,52 @@
+# Handle Untrusted Requests
+
+This document is about how to handle untrusted requests for trace and tags.
+
+## What is an untrusted request?
+An untrusted request refers to a request coming from an external untrusted source (e.g. external
+customer request).
+
+## How does OpenCensus know if a request is trusted or not?
+The OpenCensus library does not have a mechanism to determine if an incoming request is trusted 
```

Comment:
Add a paragraph explaining that the starting options should take an `isTrusted bool` option.